### ISSUE TITLE
add support to resolve OpenBiodiv taxon names

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/nomer#24]
Pensoft maintains OpenBiodiv, a biodiversity graph. In this graph, taxonomic names and associated hierarchies exist. 

Suggest to add support to resolve taxonomic hierarchies of taxonomic identifiers known to OpenBiodiv . 

Related issue https://github.com/globalbioticinteractions/globalbioticinteractions/issues/481 .

**Comment by jhpoelen** [comment]
Proposed functionality would look something like:

```
$ echo -e "http://openbiodiv.net/4B689A17-2541-4F5F-A896-6F0C2EEA3FB4" | nomer append openbiodiv 
http://openbiodiv.net/4B689A17-2541-4F5F-A896-6F0C2EEA3FB4	SAME_AS	http://openbiodiv.net/4B689A17-2541-4F5F-A896-6F0C2EEA3FB4	Acanthaceae	family		Plantae | Tracheophyta | Magnoliopsida | Lamiales | Acanthaceae		kingdom | phylum | class | order | family	http://openbiodiv.net/4B689A17-2541-4F5F-A896-6F0C2EEA3FB4	
```

**Comment by jhpoelen** [comment]
First pass at OpenBiodiv term matching made available in Nomer v0.1.18 .
